qualityassuranceevents.md

```diff
@@ -77,6 +122,22 @@ Return codes:
 
 Provide paginated list of the qa events available.
 
+### Get qualityassuranceevents created by the current user
+**GET /api/integration/qualityassuranceevents/search/findByCurrentUser?target=<:item-uuid>[&size=10&page=0]**
```

Comment:
Unless I'm missing it, we don't seem to have docs for `byTarget`?  

For example:
* `GET /api/integration/qualityassuranceevents/search/byTarget?target=[uuid]`

Reply:
it doesn't exist (at least in the corresponding REST PR), maybe you were confused by this private utility method? https://github.com/4Science/DSpace/blob/CST-12108-WithdrawnAndReinstate_topics/dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/QAEventRestRepository.java#L127